api/v1beta1/groupversion_info.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1beta1 contains API Schema definitions for the lm-eval-service v1beta1 API group
+// +kubebuilder:object:generate=true
+// +groupName=lm-eval-service.github.com
+package v1beta1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+const (
+	GroupName     = "lm-eval-service.github.com"
+	Version       = "v1beta1"
+	KindName      = "EvalJob"
+	FinalizerName = "lm-eval-service.github.com/finalizer"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "lm-eval-service.github.com", Version: "v1beta1"}
```

Comment:
I wonder if we should use something like `foundation-model-stack.github.com` as the `group` with `LMEvalJob` as the `kind`. Feels like we may have other `kinds` we want to bundle under this. I could also see not coupling this to other projects in the org. If we go that route, I'd drop the `"service"` since this is NOT managing the REST microservice.

Reply:
I will change the GroupName to `foundation-model-stack.github.com` and Kind to `LMEvalJob`. Then we can expand to other kinds for other services in the future.